pynamodb/tests/test_signals.py

```diff
@@ -0,0 +1,106 @@
+import sys
+
+import mock
+import pytest
+
+from pynamodb.connection import Connection
+from pynamodb.connection.signals import _FakeNamespace
+from pynamodb.connection.signals import pre_boto_send, post_boto_send
+
+try:
+    import blinker
+except ImportError:
+    blinker = None
+
+
+pytestmark = pytest.mark.skipif(
```

Comment:
Do you know of a good way to test the behavior without blinker being installed? I have tried to mock out `import` through `__builtins__` and `sys.modules` with no luck. From what it looks like, Python caches the modules. I have also tried using the `reload` function, but that didn't work either.

If you have ideas, please let me know.